src/next/components/Button/Button.tsx

```diff
@@ -0,0 +1,48 @@
+import { ButtonProps } from "~/common"
+import { SUPPORTED_ICONS_MAP } from "~/common/Button"
+
+// Theme specific config
+const NEXT_BUTTON_COLOR_VARIANTS = ["black", "white"] as const
+export type NextButtonColorVariant = (typeof NEXT_BUTTON_COLOR_VARIANTS)[number]
+// https://tailwindcss.com/docs/content-configuration#dynamic-class-names
+const colorVariantToClassMap: Record<NextButtonColorVariant, string> = {
+  black: "bg-[#333333] text-white",
+  white: "bg-white text-[#333333]",
+}
```

Comment:
design used `#333333` which is not within the predefined tailwind colours, closest would be `gray-700` or `gray-800`
not sure if there's a neater way to use this colour?

Reply:
Yeah we can use this for now, I think the comment can be removed as well